client/pages/play/_roomName.vue

```diff
@@ -96,6 +99,9 @@ export default {
     },
   },
   methods: {
+    focusInput() {
+      this.$refs.roomNameInput.focus();
+    },
```

Comment:
I think you can achieve the same result by putting an `autofocus` attribute on the input, did you try that?

Reply:
I did not. It may work, but I'm not sure how the browser will handle the view being shown and hidden.